acstools/utils_findsat_mrt.py

```diff
@@ -1199,10 +1200,54 @@ def update_dq(filename, ext, mask, dqval=16384, verbose=True):
         tailored for ACS/WFC.
     verbose : bool, optional
         Print extra information to the terminal.
+    expand_mask: int, optional
+        Allows the mask to be expanded to match the size of the original mask.
+        This is relevant for masks that were generated used binned versions of
+        the original image
 
     """
     with fits.open(filename, mode='update') as pf:
         dqarr = pf[ext].data
+
+        # check that the old mask and new mask are the same size
+        same_size = mask.shape == dqarr.shape
+
+        if not same_size:
+            LOG.info('Inconsistent mask sizes: \n'
+                     'Input mask: {} \n'
+                     'DQ array: {}'.format(mask.shape, dqarr.shape))
+            if not expand_mask:
+                LOG.warning('Cannot proceed due to size mismatch.\n'
+                            'Set expand_mask=True if the mask was generated '
+                            'with binned data and needs to be enlarged to the '
+                            'original size')
+                return
+            else:
+                LOG.info('Enlarging mask to original size')
+                factor_x = np.floor(dqarr.shape[1]/mask.shape[1]).astype(int)
```

Comment:
The logger saying "enlarging" but I don't see any checks on whether one dimension is really larger than the other one. Won't it be shrinking in some cases? Should that be allowed?

Reply:
Seconding this. I did a couple of tests and the error for a mask that is bigger than the DQ array is pretty cryptic. If you never expect this case to actually happen, should handle it more elegantly.

Reply:
Thanks y'all. I can't think of a situation where someone would want to supersample the original image to make it larger for satellite trail detection (feel free to chime in if you think I'm thinking too narrowly). I think the best path forward is to enforce self.binsize > 1, which will always yield a mask smaller or equal in size to the DQ array. I can add some checks inside the updatedq() function to check this as well.